login-workflow/docs/services.md

```diff
@@ -50,6 +50,12 @@ constructor(pxbAuthConfig: PxbAuthConfig) {
 -   **customEmailValidator** (optional): _`ValidatorFn`_
     -   Custom email regex requirements.
     -   Default: `undefined`
+-   **customFirstNameRequirements** (optional): _`{ maxLength: number }`_
+    -   Custom first name requirements.
+    -   Default: `{ maxLength: 1000 }`
```

Comment:
This is kinda goofy. The default should be 'no customizations' and then we only check for the limits in the component if the customFirstNameRequirements object is not undefined or empty and has a value for maxLength. Always having a max length and defaulting it to 1000 feels like a hack.